active/Expander/Expander.md

```diff
@@ -0,0 +1,131 @@
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+# Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+ > This spec corresponds to [this issue](https://github.com/microsoft/microsoft-ui-xaml/issues/3279) on the WinUI repo. 
+
+Throughout Windows, different expander controls are used by various apps and surfaces. There’s currently no consistent way to address this common UX pattern. This control is needed for situations where expanding (instead of overlaying) content is needed.  An Expander control is motivated by its use in many app scenarios and supporting developers in migrating from WPF and WCT.  
+
+# Description
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+An Expander control is a UI component that provides a standard interaction for pushing adjacent content while expanding and collapsing. An Expander is independent of the contents inside it, including controls. 
+
+## Is this the right control?
+An Expander is not intended for flyout-type behaviors where a UI component overlays on top of adjacent content. In other words, Expander will 'push' and not overlay or fly out.
+
+# Examples
+<!-- Use this section to explain the features of the API, showing
+example code with each description. The general format is: 
+  feature explanation,
+  example code
+  feature explanation,
+  example code
+  etc.-->
+  
+<!-- Code samples should be in C# and/or C++/WinRT -->
+
+<!-- As an example of this section, see the Examples section for the PasswordBox control 
+(https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
+
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+# API Notes
```

Comment:
I think we should determine which properties of [ContentControl](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.contentcontrol?view=winrt-19041) propagate into the control and in which way, e.g. how does foreground or background get propagated down to the content.

Reply:
Very good point. Specifically to the Header property (the Content property is inherited from ContentControl and not affected by Expander).

Reply:
The background and foreground of ContentControl do not propagate into the Header or Content. Background color set with ExpanderHeaderBackground is overwritten by whatever is inside the Header. How should these properties propagate from ContentControl to Expander's Content?

Reply:
The ContentControl.Foreground property does inherit (but not Background). There's other properties too, like FontSize. I think the answer is that properties only inherit to Content, but we should call it out in the description/remarks.

Reply:
Discussion: The inheritable properties _should_ inherit to both Header and Content, and likely already do in the implementation. Note that frequently controls have a default style though that sets inheritable properties as well.

Recommend: investigate and follow up, but general principle should be that inheritable properties behave the same for the Header and the Content.

Reply:
The inheritable properties such as FontSize generally don't inherit to the Header because the Header is in a ToggleButton, and as said above it has a default style that sets most/all of the inheritable properties, and that takes precedence. DataContext does inherit though.